### ISSUE TITLE
Consider adding new Assessment question about contrast in time series of removals

### ISSUE THREAD

**Issue by serenalomonico** [shcaba/FishPath-updates#176]
Previous team note: "If there is not adequate contrast in catch, are there assessments you can’t use? For Shijie’s method, if catch is flat, you can’t predict depletion."

ND: "I think we need to add a (red) caveat question to the Assessment section asking if there is adequate contrast in the time series of removals, as opposed to a constant value or “one-way trip” ".

**Comment by shcaba** [comment]
Yes, some methods do get screwed up over flat catch, which is a disturbing attribute! While I think most methods should handle this type of thing, I agree adding a question about the trend in the catch history is a good idea, and at least add a red caveat to the method that estimate stock status from catch.

**Comment by bsnouffer** [comment]
Discussed this on a call with Natalie and Jason. The answers for the fishery dependent time series question (243) have been updated to mention contrast in removals. It was decided that a further question is not needed. 

There are a couple of couple of catch only methods that no contrast in removals cause a problem. Will review to see which ones need a static caveat.

Leaving open until the review for additional static caveats is complete.
